content/boilerplates/helm-security-warning.md

```diff
@@ -0,0 +1,6 @@
+&nbsp;
+{{< warning >}}
+Please be aware the instructions for using Helm with Tiller do not use secure defaults.  Please
+reference the [Helm Security Guidelines](https://helm.sh/docs/using_helm/#securing-your-helm-installation)
```

Comment:
s/Please reference/See/ to simplify and remove repetition.
Optional: drop "Please be aware"